source/include/VirtualInstrument.h

```diff
@@ -246,17 +248,17 @@ class ClumpParseState
     InstructionAllocator* _cia;
     
     Int32           _argCount;
-    void*           _argPointers[kMaxArguments];
-    TypeRef         _argTypes[kMaxArguments];
-    
-    Int32           _argPatches[kMaxArguments];     // Arguments that need patching
+    std::vector<void*> _argPointers;
```

Comment:
It is difficult to extricate this commit from the rest because of the other variable renames and spacing fixes made.  (I tried, and after the rebase failing 4 times requiring manual edits to fix up, I gave up).   It we decide not to use stl, it would likely be an an alternate implementation behind an #if/#else anyway, or our own implementation that mimics the STL API, so this code would remain.  Therefore, I'm inclined to keep it in this PR.